CMakeLists.txt

```diff
@@ -247,7 +247,7 @@ add_option("WOLFSSL_SINGLE_THREADED"
     "no" "yes;no")
 
 # TODO: Logic here isn't complete, yet (see AX_PTHREAD)
-if(NOT WOLFSSL_SINGLE_THREADED)
+if(NOT SINGLE_THREADED)
```

Comment:
This is not the pre-processor this is the CMake argument. See `add_option("WOLFSSL_SINGLE_THREADED"` at 245. There is no issue here... 
<img width="617" alt="Screen Shot 2023-07-12 at 2 59 20 PM" src="https://github.com/wolfSSL/wolfssl/assets/2692870/89ed1970-cc57-4c98-b753-871d2c6407e7">
Please close PR.